Build.proj

```diff
@@ -10,7 +10,10 @@
   <Import Project="$(RepositoryEngineeringDir)SubsetValidation.targets" />
 
   <!-- Upfront restore hooks -->
-  <Import Project="$(RepositoryEngineeringDir)restore\docs.targets" />
+  <ItemGroup>
+    <PackageDownload Include="Microsoft.Private.Intellisense" Version="[$(MicrosoftPrivateIntellisenseVersion)]" />
+  </ItemGroup>
```

Comment:
Actually, can you please move that `PackageDownload` item into intellisense.targets? We want to inline this dependency into the project's restore phases that need it. That avoid an artificial upfront restore requirement.

Reply:
Good point. Makes sense.